completions/bash/crio

```diff
@@ -59,6 +59,7 @@ h
 --insecure-registry
 --internal-wipe
 --irqbalance-config-file
+--irqbalance-config-restore-file
```

Comment:
the commit message of this change needs to be updated to reflect the fact we now use a different special value to disable the feature